src/main/webapp/WEB-INF/views/cars/create.jsp

```diff
@@ -0,0 +1,15 @@
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<html>
+<head>
+    <title>Create Car</title>
+</head>
+<body>
+<h1>Creating car Form</h1>
+
+<form method="post" action="${pageContext.request.contextPath}/cars/create">
+  Model <input type="text" name="model"><br>
+  Manufacturer id <input type="text" name="manufacturer_id"><br>

```

Comment:
Also you can use "required" attribute in input to insure that this line will contain any value. Same for all forms in jsp pages
![image](https://user-images.githubusercontent.com/79330731/202911421-8455c349-5e42-4ad2-b393-d6e490a6ae4e.png)